signer/src/message.rs

```diff
@@ -452,6 +507,48 @@ impl wsts::net::Signable for WstsMessage {
     }
 }
 
+impl wsts::net::Signable for OutPointMessage {
+    fn hash(&self, hasher: &mut sha2::Sha256) {
+        hasher.update(self.txid);
+        hasher.update(self.vout.to_be_bytes());
+    }
+}
+
+impl wsts::net::Signable for QualifiedRequestId {
+    fn hash(&self, hasher: &mut sha2::Sha256) {
+        hasher.update(self.request_id.to_be_bytes());
+        hasher.update(self.txid.as_bytes());
+        hasher.update(self.block_hash.as_bytes());
+    }
+}
+
+impl wsts::net::Signable for TxRequestIds {
+    fn hash(&self, hasher: &mut sha2::Sha256) {
+        for deposit in &self.deposits {
+            deposit.hash(hasher);
+        }
+        for withdrawal in &self.withdrawals {
+            withdrawal.hash(hasher);
+        }
+    }
+}
+
+impl wsts::net::Signable for SbtcRequestsContextMessage {
+    fn hash(&self, hasher: &mut sha2::Sha256) {
+        hasher.update("SIGNER_NEW_BITCOIN_TX_CONTEXT");
+        for request in &self.requests {
+            request.hash(hasher);
+        }
+        hasher.update(self.fee_rate.to_be_bytes());
+        // if let Some(fee_total) = self.last_fee_total {
+        //     hasher.update(fee_total.to_be_bytes());
+        // }
+        // if let Some(fee_rate) = self.last_fee_rate {
+        //     hasher.update(fee_rate.to_be_bytes());
+        // }
```

Comment:
?

Reply:
I think we need to hash them still though. It's a shame since we'll be deleting it soonish.